aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,97 @@
+
+.. image:: overview/_static/views_solution_explorer.png
+  :height: 100
+  :width: 200
+
+In addition to action-specific views, :doc:`partial views <partial>`, :doc:`layouts, and other special view files <layout>` can be used to help reduce repetition and allow for reuse within the app's views.
+
+Benefits of Using Views
+-----------------------
+
+Views provide `separation of concerns <http://deviq.com/separation-of-concerns/>`_ within an MVC app, encapsulating user interface level markup separately from business logic. ASP.NET MVC views use :doc:`Razor syntax <razor>` to make switching between HTML markup and server side logic painless. Common, repetitive aspects of the app's user interface can easily be reused between views using :doc:`layout and shared directives <layout>` or :doc:`partial views <partial>`.
+
+Creating a View
+---------------
+
+Views that are specific to a controller are created in the *Views/[ControllerName]* folder. Views that are shared among controllers are placed in the */Views/Shared* folder. Name the view file the same as its associated controller action, and add the *.cshtml* file extension. For example, to create a view for the *About* action on the *Home* controller, you would create the *About.cshtml* file in the */Views/Home* folder.
+
+A sample view file (*About.cshtml*):
+
+.. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Views/Home/About.cshtml
+   :language: html
+
+*Razor* code is denoted by the ``@`` symbol. C# statements are run within Razor code blocks set off by curly braces (``{`` ``}``), such as the assignment of "About" to the ``ViewData["Title"]`` element shown above. Razor can be used to display values within HTML by simply referencing the value with the ``@`` symbol, as shown within the ``<h2>`` and ``<h3>`` elements above.
+
+This view focuses on just the portion of the output for which it is responsible. The rest of the page's layout, and other common aspects of the view, are specified elsewhere. Learn more about :doc:`layout and shared view logic <layout>`.
+
+How do Controllers Specify Views?
+---------------------------------
+
+Views are typically returned from actions as a :dn:cls:`~Microsoft.AspNetCore.Mvc.ViewResult`. Your action method can create and return a ``ViewResult`` directly, but more commonly if your controller inherits from :dn:cls:`~Microsoft.AspNetCore.Mvc.Controller`, you'll simply use the ``View`` helper method, as this example demonstrates:
+
+*HomeController.cs*
+
+.. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Controllers/HomeController.cs
+  :language: c#
+  :lines: 16-21
+  :emphasize-lines: 5
+  :dedent: 4
+  
+The ``View`` helper method has several overloads to make returning views easier for app developers. You can optionally specify a specific view to return, as well as a model to pass to the view. With no parameters, the default convention of using a view name that corresponds to the action name is used.
```

Comment:
Delete this sentence, it doesn't contain any useful info that elsewhere in the doc.